openff/evaluator/backends/backends.py

```diff
@@ -143,7 +143,7 @@ def __init__(
             The maximum amount of memory available to each thread.
         wallclock_time_limit: str
             The maximum amount of wall clock time that a worker can run for. This should
-            be a string of the form `HH:MM` where HH is the number of hours and MM the number
+            be a string of the form `HH:MM:SS` where HH is the number of hours and MM the number
```

Comment:
Not that important but I think you forgot to add the description for SS.

Reply:
Thank you for the review! Updated the description.